sdk/src/main/resources/io/opentelemetry/sdk/version.properties

```diff
@@ -0,0 +1 @@
+sdk.version=0.6.0-SNAPSHOT
```

Comment:
Can you delete this file? We want to generate into build at build time, not into the codebase. If you remove this and apply the suggestions and tests still pass, then you should be good